app-tracking-protection/vpn-impl/src/main/java/dummy/ui/VpnControllerActivity.kt

```diff
@@ -150,7 +150,7 @@ class VpnControllerActivity : DuckDuckGoActivity(), CoroutineScope by MainScope(
             findViewById(R.id.vpnAppTrackerCompaniesBlockedWeek)
         appTrackersBlockedWeekTextView = findViewById(R.id.vpnAppTrackersBlockedWeek)
         lastAppTrackerDomainTextView = findViewById(R.id.vpnAppLastTrackerDomain)
-
+        lastWebTrackerDomainTextView = findViewById(R.id.vpnWebLastTrackerDomain)
```

Comment:
This is used only on debug versions for stats, however it was crashing for some time and I fixed it here.